lib/init.coffee

```diff
@@ -6,13 +6,13 @@ module.exports =
     executablePath:
       title: 'Executable Path'
       type: 'string'
-      default: 'pep257'
-      description: "Path to executable pep257 cmd."
+      default: 'pydocstyle'
```

Comment:
I see. Maybe a fallback is not the right solution. Would just be good if we could avoid failing silently due to a missing executable. I had a similar problem when I tried to pass command-line options to `pep257` for ignoring test files in the "executable path" field, which made the plugin to stop complaining about issues in test files... but also in all other files, since `pep257` just couldn't be called at all anymore. So if we could generate a clear warning somewhere, that would be good. Otherwise, a note in the readme is fine, but such a thing can be easily overlooked.